comparator/src/main/java/org/mobilitydata/gtfsvalidator/outputcomparator/util/ValidationReport.java

```diff
@@ -0,0 +1,151 @@
+   * @param path the path to the json file
+   * @return the {@code ValidationReport} that contains the {@code ValidationReport} related to the
+   *     json file whose path was passed as parameter.
+   */
+  public static ValidationReport fromPath(Path path) throws IOException {
+    BufferedReader reader = Files.newBufferedReader(path);
+    ValidationReport validationReport = GSON.fromJson(reader, ValidationReport.class);
+    reader.close();
+    return validationReport;
+  }
+
+  /**
+   * Creates a {@code ValidationReport} from a json string.
+   *
+   * @param jsonString the json string
+   * @return the {@code ValidationReport} that contains the {@code ValidationReport} related to the
+   *     json string passed as parameter.
+   */
+  public static ValidationReport fromJsonString(String jsonString) {
+    return GSON.fromJson(jsonString, ValidationReport.class);
+  }
+
+  /**
+   * Returns the list of {@code NoticeAggregate} of this {@code ValidationReport}.
+   *
+   * @return the list of {@code NoticeAggregate} of this {@code ValidationReport}.
+   */
+  public List<NoticeAggregate> getNotices() {
+    return Collections.unmodifiableList(notices);
+  }
+
+  /**
+   * Returns the immutable and ordered set of error codes contained in this {@code ValidationReport}
+   *
+   * @return the immutable and ordered set of error codes contained in this {@code ValidationReport}
+   */
+  public Set<String> getErrorCodes() {
+    return Collections.unmodifiableSet(errorCodes);
+  }
+
+  /**
+   * Compares two validation reports: returns true if they contain the same set of error codes.
+   *
+   * @param otherValidationReport the other {@code ValidationReport}.
+   * @return true if the two {@code ValidationReport} contain the same set of error codes, false
+   *     otherwise.
+   */
+  public boolean hasSameErrorCodes(ValidationReport otherValidationReport) {
+    return getErrorCodes().equals(otherValidationReport.getErrorCodes());
+  }
+
+  /**
+   * Returns the number of new error codes introduced by the other {@code ValidationReport} passed
+   * as parameter, e.g. if this {@code ValidationReport} has the following error codes:
+   *
+   * <ul>
+   *   <li>invalid_phone_number;
+   *   <li>number_out_of_range;
+   * </ul>
+   *
+   * and the other {@code ValidationReport} has the following error codes:
+   *
+   * <ul>
+   *   <li>invalid_phone_number;
+   *   <li>number_out_of_range;
+   *   <li>invalid_email_address;
+   *   <li>invalid_url;
+   * </ul>
+   *
+   * <p>then this methods returns 2 as it contains two new errors codes (invalid_email_address,
+   * invalid_url) not present in this {@code ValidationReport}
+   *
+   * @param other the other {@code ValidationReport}
+   * @return the number of new error codes introduced by the other {@code ValidationReport} passed
+   *     as parameter.
+   */
+  public int getNewErrorCount(ValidationReport other) {
+    return Sets.difference(other.getErrorCodes(), getErrorCodes()).size();
```

Comment:
As discussed during our last meeting: this could be nice but not a priority now. @isabelle-dr will come back to us with more details about the possible use cases needed for these acceptance tests. 

In order to keep track of this, I will leave this discussion open for now. Will revisit in the future if needed.